lua/luasnip/nodes/optional_arg.lua

```diff
@@ -1,5 +1,7 @@
 local M = {}
 
+-- FIXME: This is not documented!
+
```

Comment:
Right, another new thing from `self-dependent-dNode`, I'll write some docs for that soon!

Reply:
Fixed in 6204bbd3e99526f766bd029773c5f19145b25ef8